db/models/Place.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose')
+
+// placeholders for now
+const categories = ['restaurant, cafe']
+const accesibilityOptions = ['braille menu']
+
+const placeSchema = mongoose.Schema(
+  {
+    ownerId: { type: mongoose.Schema.Types.ObjectId, ref: 'User' },
+    name: String,
+    description: String,
+    address: String,
+    lat: Number,
```

Comment:
@des-des it's not listed in Mongoose schema types:

>Following are all validSchema Types.
String
Number
Date
Buffer
Boolean
Mixed
Objectid
Array